test/hpe_test_utils.py

```diff
@@ -30,7 +31,10 @@ class OneViewBaseTest(object):
     @pytest.fixture(autouse=True)
     def setUp(self, mock_ansible_module, mock_ov_client, request, testing_module):
         class_name = type(self).__name__
-        marker = request.node.get_marker('resource')
+        if StrictVersion(pytest.__version__) < StrictVersion("3.6"):
```

Comment:
why we are restricting the python version for the tests?

Reply:
This line does not restrict the Python version for test, but it check the PyTest version in order to get the marker in a proper way.

Since I've installed PyTest 4, all the tests started failing due to this error: 
`AttributeError: 'Function' object has no attribute 'get_marker'`

Using this "If Section" I have been able to test either using PyTest 3.3.0 (TravisCI) and 4.0.1 (Local Build).

Reply:
thanks for correcting me. As per pytest [documentation,](https://docs.pytest.org/en/latest/mark.html#updating-code) get_marker is deprecated, so may be we should upgrade our travis.yml script to install pytest>=3.6 version. Thanks for letting us know.

Reply:
Glad to help 👍 

I came across this problem by installing PyTest using test_requirements.txt.
I haven't found how to change pytest version in travis.yml and in order to avoid fixed version inside test_requirements.txt file I have preferred to add those lines of code.

Just let me know how you want to proceed 👍

Reply:
@sijeesh - Please share your thoughts on this.

Reply:
We should update our test_requirements file with Pytest version if our tests work only with specific versions of Pytest.  But I think we can ignore it for now as this checking solves the issue.